### ISSUE TITLE
lima fails to start on a fresh OS, fresh install on M3 Pro: `Error: Image at 0013FD41000 start failed: Not Found` ... `Instruction abort: Translation fault, first level`

### ISSUE THREAD

**Issue by nbibler** [lima-vm/lima#1996]
> [!TIP]
>
> (**EDIT** by @AkihiroSuda)
>
> ### Workaround
> Specify `--vm-type=vz` in `limactl create`
> ```console
> limactl create --vm-type=vz ...
> ```
> ### Proper fix
> Expected to be fixed in Lima v0.19.1
> - https://github.com/lima-vm/lima/pull/2061

- - -
### Description

Using Homebrew to install lima on MacOS 14.1.1 (23B2082), Apple M3 Pro, the installation completes successfully, but starting up lima freezes at "INFO[0000] [hostagent] Waiting for the essential requirement 1 of 5: "ssh"" and eventually fails. The log files seem to indicate some type of build error: "Error: Image at 0013FD41000 start failed: Not Found", "Instruction abort: Translation fault, first level".

```
$ brew info lima
==> lima: stable 0.18.0 (bottled), HEAD
Linux virtual machines
https://github.com/lima-vm/lima
/opt/homebrew/Cellar/lima/0.18.0 (104 files, 169.8MB) *
  Poured from bottle using the formulae.brew.sh API on 2023-11-09 at 16:21:16
From: https://github.com/Homebrew/homebrew-core/blob/HEAD/Formula/l/lima.rb
License: Apache-2.0
```

```
$ brew install lima

$ limactl create --tty=false
INFO[0020] Attempting to download the image              arch=aarch64 digest="sha256:78d35a2f551d281912ee7e5202660d94d713aa1b5de86a17e261328cc2f093d4" location="https://cloud-images.ubuntu.com/releases/23.10/release-20231011/ubuntu-23.10-server-cloudimg-arm64.img"
INFO[0062] Downloaded the image from "https://cloud-images.ubuntu.com/releases/23.10/release-20231011/ubuntu-23.10-server-cloudimg-arm64.img" 
INFO[0062] Attempting to download the nerdctl archive    arch=aarch64 digest="sha256:ea30ab544c057e3a0457194ecd273ffbce58067de534bdfaffe4edf3a4da6357" location="https://github.com/containerd/nerdctl/releases/download/v1.6.2/nerdctl-full-1.6.2-linux-arm64.tar.gz"
INFO[0076] Downloaded the nerdctl archive from "https://github.com/containerd/nerdctl/releases/download/v1.6.2/nerdctl-full-1.6.2-linux-arm64.tar.gz" 
INFO[0076] Run `limactl start default` to start the instance. 

$ limactl start default
INFO[0000] Using the existing instance "default"        
INFO[0000] QEMU binary "/opt/homebrew/bin/qemu-system-aarch64" seems properly signed with the "com.apple.security.hypervisor" entitlement 
INFO[0000] [hostagent] hostagent socket created at /Users/nbibler/.lima/default/ha.sock 
INFO[0000] [hostagent] Starting QEMU (hint: to watch the boot progress, see "/Users/nbibler/.lima/default/serial*.log") 
INFO[0000] SSH Local Port: 60022                        
INFO[0000] [hostagent] Waiting for the essential requirement 1 of 5: "ssh" 
```

Checking the logs (serial.log):

```
UEFI firmware (version edk2-stable202302-for-qemu built at 17:14:55 on Mar 15 2023)
Error: Image at 0013FD41000 start failed: Not Found


Synchronous Exception at 0x000000013FD37E08
PC 0x00013FD37E08 (0x00013FD35000+0x00002E08) [ 0] ArmCpuDxe.dll
PC 0x00013FD37E08 (0x00013FD35000+0x00002E08) [ 0] ArmCpuDxe.dll
PC 0x00013FD37D00 (0x00013FD35000+0x00002D00) [ 0] ArmCpuDxe.dll
PC 0x00013FD380BC (0x00013FD35000+0x000030BC) [ 0] ArmCpuDxe.dll
PC 0x0000476C36C0 (0x0000476BB000+0x000086C0) [ 1] DxeCore.dll
PC 0x0000476C8938 (0x0000476BB000+0x0000D938) [ 1] DxeCore.dll
PC 0x0000476C2F44 (0x0000476BB000+0x00007F44) [ 1] DxeCore.dll
PC 0x00013FD38DA0 (0x00013FD35000+0x00003DA0) [ 2] ArmCpuDxe.dll
PC 0x0000476C2848 (0x0000476BB000+0x00007848) [ 3] DxeCore.dll
PC 0x0000476CFF84 (0x0000476BB000+0x00014F84) [ 3] DxeCore.dll
PC 0x0000476C6340 (0x0000476BB000+0x0000B340) [ 3] DxeCore.dll

[ 0] /home/kraxel/projects/qemu/roms/Build/ArmVirtQemu-AARCH64/DEBUG_GCC5/AARCH64/ArmPkg/Drivers/CpuDxe/CpuDxe/DEBUG/ArmCpuDxe.dll
[ 1] /home/kraxel/projects/qemu/roms/Build/ArmVirtQemu-AARCH64/DEBUG_GCC5/AARCH64/MdeModulePkg/Core/Dxe/DxeMain/DEBUG/DxeCore.dll
[ 2] /home/kraxel/projects/qemu/roms/Build/ArmVirtQemu-AARCH64/DEBUG_GCC5/AARCH64/ArmPkg/Drivers/CpuDxe/CpuDxe/DEBUG/ArmCpuDxe.dll
[ 3] /home/kraxel/projects/qemu/roms/Build/ArmVirtQemu-AARCH64/DEBUG_GCC5/AARCH64/MdeModulePkg/Core/Dxe/DxeMain/DEBUG/DxeCore.dll

  X0 0x0000000047FFE020   X1 0x000000013F0D7003   X2 0x0000000000100000   X3 0x0000000000000000
  X4 0x0000000000000200   X5 0x0000000000000004   X6 0x0060000000000000   X7 0xFF9F000000000F3F
  X8 0x000000013FFFF008   X9 0x0000000400000000  X10 0x000000013F0D5000  X11 0x000000013FA37FFF
 X12 0x0000000000000000  X13 0x0000000000000008  X14 0x0000000000000000  X15 0x0000000000000000
 X16 0x000000013FD36208  X17 0x0000000047702000  X18 0x0000000000000000  X19 0x0000000100000000
 X20 0x0000000047FFE000  X21 0x0000000000000001  X22 0x0000000000000020  X23 0x000000013F0D7003
 X24 0x000000003FFFFFFF  X25 0x0000000000000000  X26 0x000000013CAA0000  X27 0x0000000000000002
 X28 0x000000013FD3D020   FP 0x00000000476BA8A0   LR 0x000000013FD37E08  

  V0 0x0000000000000000 0000000000000000   V1 0x0000000000000000 0000000000000000
  V2 0x0000000000000000 0000000000000000   V3 0x0000000000000000 0000000000000000
  V4 0x0000000000000000 0000000000000000   V5 0x0000000000000000 0000000000000000
  V6 0x0000000000000000 0000000000000000   V7 0x0000000000000000 0000000000000000
  V8 0x0000000000000000 0000000000000000   V9 0x0000000000000000 0000000000000000
 V10 0x0000000000000000 0000000000000000  V11 0x0000000000000000 0000000000000000
 V12 0x0000000000000000 0000000000000000  V13 0x0000000000000000 0000000000000000
 V14 0x0000000000000000 0000000000000000  V15 0x0000000000000000 0000000000000000
 V16 0x0000000000000000 0000000000000000  V17 0x0000000000000000 0000000000000000
 V18 0x0000000000000000 0000000000000000  V19 0x0000000000000000 0000000000000000
 V20 0x0000000000000000 0000000000000000  V21 0x0000000000000000 0000000000000000
 V22 0x0000000000000000 0000000000000000  V23 0x0000000000000000 0000000000000000
 V24 0x0000000000000000 0000000000000000  V25 0x0000000000000000 0000000000000000
 V26 0x0000000000000000 0000000000000000  V27 0x0000000000000000 0000000000000000
 V28 0x0000000000000000 0000000000000000  V29 0x0000000000000000 0000000000000000
 V30 0x0000000000000000 0000000000000000  V31 0x0000000000000000 0000000000000000

  SP 0x00000000476BA8A0  ELR 0x000000013FD37E08  SPSR 0x80000205  FPSR 0x00000000
 ESR 0x86000005          FAR 0x000000013FD37E08

 ESR : EC 0x21  IL 0x1  ISS 0x00000005

Instruction abort: Translation fault, first level

Stack dump:
  00000476BA7A0: 000000013F0D7000 000000013F0D54F8 0060000000000000 FF9F000000000F3F
  00000476BA7C0: 000000013CAA0000 000000013CAA0000 000000003CA00000 0000000C00000001
  00000476BA7E0: 0000000000000002 0000000000000F28 00000000476BA8A0 000000013FD37D00
  00000476BA800: 0000000100000000 0000000047FFE000 0000000000000001 0000000000000020
  00000476BA820: 000000013F0D7000 000000003FFFFFFF 000000013F0D7000 000000013CAA0000
  00000476BA840: 0000000000000002 000000013FD3D020 0000000047FFD000 000000013F0D7F28
  00000476BA860: 0060000000000000 FF9F000000000F3F 000000013CC00000 000000013CAA0000
  00000476BA880: 0000000000000000 0000001500000001 0000000000000001 000000013F0D5000
> 00000476BA8A0: 00000000476BA950 000000013FD37D00 0000000047FFA000 0000000047FFF000
  00000476BA8C0: 0000000000000000 0000000000000000 0000000047FFE000 0000007FFFFFFFFF
  00000476BA8E0: 000000013F0D8228 000000013CAA0000 0000000000000001 000000013FD3D020
  00000476BA900: 00000000476BA940 0000000047FFE020 0060000000000000 FF9F000000000F3F
  00000476BA920: 0000000140000000 000000013CAA0000 0000000147FFA000 0000001E00000001
  00000476BA940: 0060000000000000 000000013F0D7000 00000000476BAA00 000000013FD380BC
  00000476BA960: 002000000000041C 0000000047FFA000 0000000000004000 00000000F4AA6000
  00000476BA980: 0000000047FFF000 0000000000000001 000000013F0D8228 00000000476E5648
ASSERT [ArmCpuDxe] /home/kraxel/projects/qemu/roms/edk2/ArmPkg/Library/DefaultExceptionHandlerLib/AArch64/DefaultExceptionHandler.c(333): ((BOOLEAN)(0==1))
```

**Comment by AkihiroSuda** [comment]
- Is this apparently specific to M3? Or macOS 14?
- Does --vm-type=vz work?

**Comment by nbibler** [comment]
@AkihiroSuda : `--vm-type=vz` does appear to work! ❤️ 

I can't easily downgrade any of my machines to test on MacOS <14. And the only Apple Silicon I have is an M3. So, I can't give much useful information to the other questions.

Thank you! I didn't even see that `--vm-type` was an option to try. 😄

**Comment by ethanzway** [comment]
m3 pro same problem
serial.log report error "Error: Image at 0013FD41000 start failed: Not Found"

**Comment by AkihiroSuda** [comment]
@ethanzway Could you report to https://gitlab.com/qemu-project/qemu/-/issues ?
Is this likely to be specific to M3 (not M2) ?

**Comment by jandubois** [comment]
https://github.com/rancher-sandbox/rancher-desktop/issues/5943 is another report of this issue with an M3 Max chip:

```
UEFI firmware (version edk2-stable202302-for-qemu built at 17:14:55 on Mar 15 2023)
Error: Image at 001BFD41000 start failed: Not Found
```

@AkihiroSuda I realize that this seems to be an upstream problem, but shouldn't we keep this issue here open until it is resolved in QEMU, and we have information on which version is required to fix it?

At least it makes it obvious to other people running into this problem that it is still unresolved.

**Comment by AkihiroSuda** [comment]
Linking the issue on the QEMU repo:
- https://gitlab.com/qemu-project/qemu/-/issues/1990

**Comment by ardbiesheuvel** [comment]
The bundled firmware image lacks the following patch 

5ce29ae84db3 ArmPkg/ArmMmuLib AARCH64: Add missing ISB after page table update

which addresses an issue that resulted in very similar symptoms on a different system.

Please confirm whether the issue is reproducible with this firmware image

https://snapshots.linaro.org/components/kernel/leg-virt-tianocore-edk2-upstream/latest/QEMU-AARCH64/RELEASE_CLANGDWARF/QEMU_EFI.fd

**Comment by AkihiroSuda** [comment]
Opened a Homebrew PR as an interim workaround until the QEMU upstream bumps up EDK2:
- https://github.com/Homebrew/homebrew-core/pull/155388

**Comment by AkihiroSuda** [comment]
Can anybody try the current master? (`brew install --HEAD qemu`)

https://gitlab.com/qemu-project/qemu/-/issues/1990#note_1673436549

**Comment by paralin** [comment]
@AkihiroSuda with `brew install --HEAD qemu` I still get:

```
Instruction abort: Permission fault, third level

[snip]

Synchronous Exception at 0x000000013C564000
ASSERT [ArmCpuDxe] /home/kraxel/projects/qemu/roms/edk2/ArmPkg/Library/DefaultExceptionHandlerLib/AArch64/DefaultExceptionHandler.c(343): ((BOOLEAN)(0==1))
```

**Comment by mprimeaux** [comment]
>  Is this likely to be specific to M3 (not M2) ?

@AkihiroSuda I'm not sure if your question was directly answered but I am unable to reproduce this issue on my M1 or M2 machines but can reproduce on my M3.  

What's a bit unusual with the M3 is that I can get Lima to successfully boot 1 per approximately every 5 attempts.

**Comment by M4urici0GM** [comment]
I was able to install with --HEAD on m3 pro

**Comment by ardbiesheuvel** [comment]
> What's a bit unusual with the M3 is that I can get Lima to successfully boot 1 per approximately every 5 attempts.

This is actually quite symptomatic for issues due to missing barrier instructions, such as the one in question.

**Comment by AkihiroSuda** [comment]
Can anybody try this on M3?
```
# The binary is from https://gitlab.com/qemu-project/qemu/-/issues/1990#note_1677624089
curl -OfSL http://files.workofard.com/QEMU_EFI.fd.gz
gunzip QEMU_EFI.fd.gz
cp QEMU_EFI.fd /opt/homebrew/share/qemu/edk2-aarch64-code.fd
# The cfg key is subject to change, follow the discussion in https://edk2.groups.io/g/devel/message/112027
export QEMU_SYSTEM_AARCH64="qemu-system-aarch64 -fw_cfg opt/org.tianocore/DisableMemAttrProtocol,string=y"
limactl start
```

This should work on M1 too.
(I tested on M2 Pro)

**Comment by eightnoteight** [comment]
> Can anybody try this on M3?

got same issue, tried it on m3, worked after following above commands

#### M3
![image](https://github.com/lima-vm/lima/assets/36945389/4a89f8ae-855e-475f-9c0f-e892461d463c)

#### Before
```
limactl start default
INFO[0000] Using the existing instance "default"
INFO[0000] Starting the instance "default" with VM driver "qemu"
INFO[0000] QEMU binary "/opt/homebrew/bin/qemu-system-aarch64" seems properly signed with the "com.apple.security.hypervisor" entitlement
INFO[0000] [hostagent] hostagent socket created at /Users/srinivas/.lima/default/ha.sock
INFO[0001] [hostagent] Starting QEMU (hint: to watch the boot progress, see "/Users/srinivas/.lima/default/serial*.log")
INFO[0001] SSH Local Port: 60022
INFO[0001] [hostagent] Waiting for the essential requirement 1 of 4: "ssh"


INFO[0086] [hostagent] Waiting for the essential requirement 1 of 4: "ssh"
^C
```

```
UEFI firmware (version edk2-stable202302-for-qemu built at 17:14:55 on Mar 15 2023)
Error: Image at 0043FD41000 start failed: Not Found


Synchronous Exception at 0x000000043FD37E08
PC 0x00043FD37E08 (0x00043FD35000+0x00002E08) [ 0] ArmCpuDxe.dll
PC 0x00043FD37E08 (0x00043FD35000+0x00002E08) [ 0] ArmCpuDxe.dll
PC 0x00043FD37D00 (0x00043FD35000+0x00002D00) [ 0] ArmCpuDxe.dll
PC 0x00043FD380BC (0x00043FD35000+0x000030BC) [ 0] ArmCpuDxe.dll
PC 0x0000476C36C0 (0x0000476BB000+0x000086C0) [ 1] DxeCore.dll
PC 0x0000476C8938 (0x0000476BB000+0x0000D938) [ 1] DxeCore.dll
PC 0x0000476C2F44 (0x0000476BB000+0x00007F44) [ 1] DxeCore.dll
PC 0x00043FD38DA0 (0x00043FD35000+0x00003DA0) [ 2] ArmCpuDxe.dll
PC 0x0000476C2848 (0x0000476BB000+0x00007848) [ 3] DxeCore.dll
PC 0x0000476CFF84 (0x0000476BB000+0x00014F84) [ 3] DxeCore.dll
PC 0x0000476C6340 (0x0000476BB000+0x0000B340) [ 3] DxeCore.dll

[ 0] /home/kraxel/projects/qemu/roms/Build/ArmVirtQemu-AARCH64/DEBUG_GCC5/AARCH64/ArmPkg/Drivers/CpuDxe/CpuDxe/DEBUG/ArmCpuDxe.dll
[ 1] /home/kraxel/projects/qemu/roms/Build/ArmVirtQemu-AARCH64/DEBUG_GCC5/AARCH64/MdeModulePkg/Core/Dxe/DxeMain/DEBUG/DxeCore.dll
[ 2] /home/kraxel/projects/qemu/roms/Build/ArmVirtQemu-AARCH64/DEBUG_GCC5/AARCH64/ArmPkg/Drivers/CpuDxe/CpuDxe/DEBUG/ArmCpuDxe.dll
[ 3] /home/kraxel/projects/qemu/roms/Build/ArmVirtQemu-AARCH64/DEBUG_GCC5/AARCH64/MdeModulePkg/Core/Dxe/DxeMain/DEBUG/DxeCore.dll

  X0 0x0000000047FFE080   X1 0x000000043F0D7003   X2 0x0000000000400000   X3 0x0000000000000000
  X4 0x0000000000000200   X5 0x0000000000000004   X6 0x0060000000000000   X7 0xFF9F000000000F3F
  X8 0x000000043FFFF008   X9 0x0000000400000000  X10 0x000000043F0D5000  X11 0x000000043FA37FFF
 X12 0x0000000000000000  X13 0x0000000000000008  X14 0x0000000000000000  X15 0x0000000000000000
 X16 0x000000043FD36208  X17 0x0000000047702000  X18 0x0000000000000000  X19 0x0000000400000000
 X20 0x0000000047FFE000  X21 0x0000000000000001  X22 0x0000000000000080  X23 0x000000043F0D7003
 X24 0x000000003FFFFFFF  X25 0x0000000000000000  X26 0x000000043CAA0000  X27 0x0000000000000002
 X28 0x000000043FD3D020   FP 0x00000000476BA8A0   LR 0x000000043FD37E08

  V0 0x0000000000000000 0000000000000000   V1 0x0000000000000000 0000000000000000
  V2 0x0000000000000000 0000000000000000   V3 0x0000000000000000 0000000000000000
  V4 0x0000000000000000 0000000000000000   V5 0x0000000000000000 0000000000000000
  V6 0x0000000000000000 0000000000000000   V7 0x0000000000000000 0000000000000000
  V8 0x0000000000000000 0000000000000000   V9 0x0000000000000000 0000000000000000
 V10 0x0000000000000000 0000000000000000  V11 0x0000000000000000 0000000000000000
 V12 0x0000000000000000 0000000000000000  V13 0x0000000000000000 0000000000000000
 V14 0x0000000000000000 0000000000000000  V15 0x0000000000000000 0000000000000000
 V16 0x0000000000000000 0000000000000000  V17 0x0000000000000000 0000000000000000
 V18 0x0000000000000000 0000000000000000  V19 0x0000000000000000 0000000000000000
 V20 0x0000000000000000 0000000000000000  V21 0x0000000000000000 0000000000000000
 V22 0x0000000000000000 0000000000000000  V23 0x0000000000000000 0000000000000000
 V24 0x0000000000000000 0000000000000000  V25 0x0000000000000000 0000000000000000
 V26 0x0000000000000000 0000000000000000  V27 0x0000000000000000 0000000000000000
 V28 0x0000000000000000 0000000000000000  V29 0x0000000000000000 0000000000000000
 V30 0x0000000000000000 0000000000000000  V31 0x0000000000000000 0000000000000000

  SP 0x00000000476BA8A0  ELR 0x000000043FD37E08  SPSR 0x80000205  FPSR 0x00000000
 ESR 0x86000005          FAR 0x000000043FD37E08

 ESR : EC 0x21  IL 0x1  ISS 0x00000005

Instruction abort: Translation fault, first level

Stack dump:
  00000476BA7A0: 000000043F0D7000 000000043F0D54F8 0060000000000000 FF9F000000000F3F
  00000476BA7C0: 000000043CAA0000 000000043CAA0000 000000003CA00000 0000000C00000001
  00000476BA7E0: 0000000000000002 0000000000000F28 00000000476BA8A0 000000043FD37D00
  00000476BA800: 0000000400000000 0000000047FFE000 0000000000000001 0000000000000080
  00000476BA820: 000000043F0D7000 000000003FFFFFFF 000000043F0D7000 000000043CAA0000
  00000476BA840: 0000000000000002 000000043FD3D020 0000000047FFD000 000000043F0D7F28
  00000476BA860: 0060000000000000 FF9F000000000F3F 000000043CC00000 000000043CAA0000
  00000476BA880: 0000000000000000 0000001500000001 0000000000000001 000000043F0D5000
> 00000476BA8A0: 00000000476BA950 000000043FD37D00 0000000047FFA000 0000000047FFF000
  00000476BA8C0: 0000000000000000 0000000000000000 0000000047FFE000 0000007FFFFFFFFF
  00000476BA8E0: 000000043F0D8228 000000043CAA0000 0000000000000001 000000043FD3D020
  00000476BA900: 00000000476BA940 0000000047FFE080 0060000000000000 FF9F000000000F3F
  00000476BA920: 0000000440000000 000000043CAA0000 0000000147FFA000 0000001E00000001
  00000476BA940: 0060000000000000 000000043F0D7000 00000000476BAA00 000000043FD380BC
  00000476BA960: 002000000000041C 0000000047FFA000 0000000000004000 00000003F4AA6000
  00000476BA980: 0000000047FFF000 0000000000000001 000000043F0D8228 00000000476E5648
ASSERT [ArmCpuDxe] /home/kraxel/projects/qemu/roms/edk2/ArmPkg/Library/DefaultExceptionHandlerLib/AArch64/DefaultExceptionHandler.c(333): ((BOOLEAN)(0==1))
```

#### After
```
cp QEMU_EFI.fd /opt/homebrew/share/qemu/edk2-aarch64-code.fd
export QEMU_SYSTEM_AARCH64="qemu-system-aarch64 -fw_cfg opt/org.tianocore/DisableMemAttrProtocol,string=y"

limactl start default
INFO[0000] Using the existing instance "default"
INFO[0000] Starting the instance "default" with VM driver "qemu"
WARN[0000] Specifying args ([-fw_cfg opt/org.tianocore/DisableMemAttrProtocol,string=y]) via $QEMU_SYSTEM_AARCH64 is supported only for debugging!
INFO[0000] QEMU binary "/opt/homebrew/bin/qemu-system-aarch64" seems properly signed with the "com.apple.security.hypervisor" entitlement
INFO[0000] [hostagent] hostagent socket created at /Users/srinivas/.lima/default/ha.sock
WARN[0000] [hostagent] Specifying args ([-fw_cfg opt/org.tianocore/DisableMemAttrProtocol,string=y]) via $QEMU_SYSTEM_AARCH64 is supported only for debugging!
INFO[0000] [hostagent] Starting QEMU (hint: to watch the boot progress, see "/Users/srinivas/.lima/default/serial*.log")
INFO[0000] SSH Local Port: 60022
INFO[0000] [hostagent] Waiting for the essential requirement 1 of 4: "ssh"
INFO[0017] [hostagent] Waiting for the essential requirement 1 of 4: "ssh"
INFO[0018] [hostagent] The essential requirement 1 of 4 is satisfied
INFO[0018] [hostagent] Waiting for the essential requirement 2 of 4: "user session is ready for ssh"
INFO[0018] [hostagent] The essential requirement 2 of 4 is satisfied
INFO[0018] [hostagent] Waiting for the essential requirement 3 of 4: "sshfs binary to be installed"
INFO[0018] [hostagent] The essential requirement 3 of 4 is satisfied
INFO[0018] [hostagent] Waiting for the essential requirement 4 of 4: "/etc/fuse.conf (/etc/fuse3.conf) to contain \"user_allow_other\""
INFO[0018] [hostagent] The essential requirement 4 of 4 is satisfied
INFO[0018] [hostagent] Mounting "/Users/srinivas" on "/Users/srinivas"
INFO[0018] [hostagent] Mounting "/tmp/lima" on "/tmp/lima"
INFO[0018] [hostagent] Mounting "/Volumes/workspace/lima-vm" on "/Volumes/workspace/lima-vm"
INFO[0018] [hostagent] Waiting for the optional requirement 1 of 2: "systemd must be available"
INFO[0018] [hostagent] Guest agent is running
INFO[0018] [hostagent] The optional requirement 1 of 2 is satisfied
INFO[0018] [hostagent] Not forwarding TCP 127.0.0.53:53
INFO[0018] [hostagent] Waiting for the optional requirement 2 of 2: "containerd binaries to be installed"
INFO[0018] [hostagent] Not forwarding TCP 0.0.0.0:22
INFO[0018] [hostagent] Forwarding TCP from 0.0.0.0:8081 to 127.0.0.1:8081
INFO[0018] [hostagent] Forwarding TCP from 0.0.0.0:5432 to 127.0.0.1:5432
INFO[0018] [hostagent] Not forwarding TCP [::]:22
INFO[0018] [hostagent] Forwarding TCP from [::]:8081 to 127.0.0.1:8081
WARN[0018] [hostagent] failed to set up forwarding tcp port 8081 (negligible if already forwarded)
INFO[0018] [hostagent] Forwarding TCP from [::]:5432 to 127.0.0.1:5432
INFO[0018] [hostagent] The optional requirement 2 of 2 is satisfied
INFO[0018] [hostagent] Waiting for the guest agent to be running
INFO[0018] [hostagent] Waiting for the final requirement 1 of 1: "boot scripts must have finished"
WARN[0018] [hostagent] failed to set up forwarding tcp port 5432 (negligible if already forwarded)
INFO[0018] [hostagent] The final requirement 1 of 1 is satisfied
INFO[0018] READY. Run `lima` to open the shell.
```

**Comment by AkihiroSuda** [comment]
The issue is expected to be fixed in Lima v0.19.1
- https://github.com/lima-vm/lima/pull/2061#issuecomment-1855308417
